application/wlp-web-java-spring/src/test/java/webapp/tier/db/PostgresServiceTest.java

```diff
@@ -80,6 +84,34 @@ void testinsert() {
 		}
 	}
 
+	@Test
+	void testselect() {
+		PostgresService svc = new PostgresService() {
+			public Connection getConnection() throws NamingException, SQLException {
+				ResultSet rs = mock(ResultSet.class);
```

Comment:
Codacy found an issue: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6218843)